apps/schema/src/components/ColumnEditModal.vue

```diff
@@ -73,7 +73,7 @@
                 />
                 <InputSelect
                   id="column_refSchema"
-                  v-model="column.refSchema"
+                  v-model="column.refSchemaName"
                   :options="schemaNames"
```

Comment:
Unrelated to the change, but I cannot find where schemaNames is defined. It seems to be a dead property. Any ideas about this?

Reply:
indeed, should be schemaIds